src/AccessibilityInsights.Extensions.AzureDevOps/ForcedDependencies.cs

```diff
@@ -33,7 +33,6 @@ private static void TestEnforcedDependencies()
         private static void ManuallyEnforcedDependencies()
         {
             // Types that we can't detect via unit tests. Namespace generally matches the package.
-            ForceDependency(typeof(Microsoft.IdentityModel.Clients.ActiveDirectory.AdalClaimChallengeException));
```

Comment:
Is there a replacement type that needs to be added here?

Reply:
Good question. I confirmed that all of the files in the ADO extension make it into the MSI file, so I think we're ok